docs/fr/docs/contributing.md

```diff
@@ -0,0 +1,501 @@
+
+```console
+$ python ./scripts/docs.py live
+
+<span style="color: green;">[INFO]</span> Serving on http://127.0.0.1:8008
+<span style="color: green;">[INFO]</span> Start watching changes
+<span style="color: green;">[INFO]</span> Start detecting changes
+```
+
+</div>
+
+Il servira la documentation sur `http://127.0.0.1:8008`.
+
+De cette façon, vous pouvez modifier la documentation/les fichiers sources et voir les changements en direct.
+
+#### Typer CLI (facultatif)
+
+Les instructions ici vous montrent comment utiliser le script à `./scripts/docs.py` avec le programme `python` directement.
+
+Mais vous pouvez également utiliser <a href="https://typer.tiangolo.com/typer-cli/" class="external-link" target="_blank">Typer CLI</a>, et vous obtiendrez l'auto-complétion dans votre terminal pour les commandes après l'achèvement de  l'installation.
+
+Si vous installez Typer CLI, vous pouvez installer la complétion avec :
+
+<div class="termy">
+
+```console
+$ typer --install-completion
+
+zsh completion installed in /home/user/.bashrc.
+Completion will take effect once you restart the terminal.
+```
+
+</div>
+
+### Apps et documentation en même temps
+
+Si vous exécutez les exemples avec, par exemple :
+
+<div class="termy">
+
+```console
+$ uvicorn tutorial001:app --reload
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+```
+
+</div>
+
+Comme Uvicorn utilisera par défaut le port `8000`, la documentation sur le port `8008` n'entrera pas en conflit.
+
+### Traductions
+
+L'aide aux traductions est TRÈS appréciée ! Et cela ne peut se faire sans l'aide de la communauté. 🌎 🚀
+
+Voici les étapes à suivre pour aider à la traduction.
+
+#### Conseils et lignes directrices
+
+* Vérifiez les <a href="https://github.com/tiangolo/fastapi/pulls" class="external-link" target="_blank">pull requests existantes</a> pour votre langue et ajouter des reviews demandant des changements ou les approuvant.
+
+!!! tip
+    Vous pouvez <a href="https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request" class="external-link" target="_blank">ajouter des commentaires avec des suggestions de changement</a> aux pull requests existantes.
+
+    Consultez les documents concernant <a href="https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews" class="external-link" target="_blank">l'ajout d'un review de pull request</a> pour l'approuver ou demander des modifications.
+
+* Vérifiez dans <a href="https://github.com/tiangolo/fastapi/issues" class="external-link" target="_blank">issues</a> pour voir s'il y a une personne qui coordonne les traductions pour votre langue.
+
+* Ajoutez une seule pull request par page traduite. Il sera ainsi beaucoup plus facile pour les autres de l'examiner.
+
+Pour les langues que je ne parle pas, je vais attendre que plusieurs autres reviews de la traduction avant de merge.
```

Comment:
I think it should be "je vais attendre plusieurs autres reviews de la traduction"

Reply:
```suggestion
Pour les langues que je ne parle pas, je vais attendre plusieurs autres reviews de la traduction avant de merge.
```